### ISSUE TITLE
Understand Impact to Baselines Relative to JAB Dissolution

### ISSUE THREAD

**Issue by brian-ruf** [GSA/automate.fedramp.gov#90]
As a consumer of the FedRAMP 800-53 Revision 5 baselines, and creator of automation in support of those baselines, I need to understand if those baselines will change to reflect the dissolution of the JAB. 

There are 20 references to the JAB in the High baseline. Most are easily addressed/ignored as they are statements similar to "must be approved by the JAB/AO" and clearly we can just drop "JAB" in these instances. 

However, there are a few instances where pursuing/maintaining a "JAB authorization" carry different requirements. These need revision/clarification. 

Below is a more complete summary based on the High baseline:

| Control | Nature of JAB Reference |
| :---: | :--- |
| AC-2(3) | "JAB/AO" reference |
| AC-8 | "JAB/AO" references (multiple) |
| AU-2 | "JAB/AO" references (multiple) |
| AU-6 | "JAB/AO" reference |
| AU-6(6) | "JAB/AO" reference |
| CA-2(1) | "Requirement: For JAB Authorization, must use an accredited 3PAO" |
| CA-2(3) | "JAB/AO" reference |
| CA-3 (c)  | "JAB/AO" reference |
| CA-6 (e) | "JAB/AO" reference |
| CA-7 (g) | "JAB/AO" reference |
| CA-7 | RE: Collaborative ConMon "It does not apply to CSOs authorized via the JAB path because the JAB performs ConMon oversight." |
| CM-2 (b) (2) | "to include when directed by the JAB" |
| CM-3 | "JAB/AO" reference |
| CP-2 | "For JAB authorizations the contingency lists include designated FedRAMP personnel." |
| CP-4 | "JAB/AO" reference |
| IR-3-2 | "JAB/AO" references (multiple) |
| MP-5 | "JAB/AO" reference |
| RA-3 | "for JAB authorizations to include FedRAMP."  |
| RA-5 (e) | "for JAB authorizations to include FedRAMP" |
| SC-7(5) | "For JAB Authorization, CSPs shall include details of this control in their Architecture Briefing"  |

**Comment by brian-ruf** [comment]
I recommend the following course of action:
- CA-2(1): the requirement vs recommendation of an accredited 3PAO must be revisited by the PMO for each authorization type and published. Until that decision is made, consider dropping this language as not applicable. 
- CA-7: the PMO must clarify the approach to ConMon oversight going forward and update this control accordingly. Until that clarity is available, consider revising this control to indicate that ConMon oversight is as directed by applicable AOs.
- CP-2: The PMO must determine when it is appropriate to have PMO POC(s) listed in a CSP's contingency plan and update this control accordingly. Until that decision is made, consider revising this language to indicate that FedRAMP personnel should be included in contingency lists as directed by applicable AOs. 
- RA-3, RA-5(3): The PMO must decide what they should be included in notifications/reporting and update these controls accordingly. Until that decision is made, consider revising this language to indicate that FedRAMP should be included as directed by applicable AOs.
- Simply adjust "JAB/AO to "AO" for all remaining controls.

**Comment by aj-stein-gsa** [comment]
> I recommend the following course of action:
> 
> * CA-2(1): ...  consider dropping this language as not applicable.

We can follow up with FedRAMP leadership to confirm, but we have [already publicly communicated a transition period](https://www.fedramp.gov/2024-08-12-moving-to-one-fedramp-authorization-an-update-on-the-jab-transition/), so re your recommendation, would removing them immediately not cause some more confusion if we do not identify the transitional state and, after the transitional state is complete, finally remove with clear indication it is deprecated?

(I like your recommendations, but re authorization type in OSCAL, I actually decided in a PR we should leave the JAB type for this very reason without expressly saying so, but it is on me for not creating an issue for the review to possibly change it holistically then, so we should call me out on that. 😅 A quick search indicates that came up really with LiSaaS and JAB by implication in https://github.com/GSA/fedramp-automation/pull/610#discussion_r1736975697 for posterity. At least you are keeping up accountable to take care of this now!)

**Comment by brian-ruf** [comment]
@aj-stein-gsa I should have been clear that publication of any removal should be timed with the JAB transition period. I did not intend to suggest anything that undermined that transition period.

**Comment by aj-stein-gsa** [comment]
> @aj-stein-gsa I should have been clear that publication of any removal should be timed with the JAB transition period. I did not intend to suggest anything that undermined that transition period.

I don't think you did, just wanted to understand mechanically how you would see the working. I do not personally know the details of a timeline or game plan for the transition on the finer point details. We will discuss internally together and find out soon enough I am sure. 🤞

**Comment by david-waltermire** [comment]
Let's create a PR, which we can have the review team review.

Here is a rough sketch of the changes.

- In place of "JAB/AO" we should use "designated lead".
- In place of "to include when directed by the JAB" use "to include when directed by the designated lead"
- For "Requirement: For JAB Authorization, must use an accredited 3PAO" use "Requirement: Must use an accredited 3PAO, unless otherwise approved by the designated lead"
- CA-7 statement should be dropped.
- Remove "for JAB authorizations to include FedRAMP"
- "For JAB Authorization, CSPs shall include details of this control in their Architecture Briefing" change to "CSPs shall include details of this control in their Architecture Briefing"

**Comment by david-waltermire** [comment]
We should wrap up https://github.com/GSA/fedramp-automation/pull/604 and then make a PR with these changes. We need to run the PR by Sam, John, and Ryan Palmer before merging to ensure we are aligned.

**Comment by Rene2mt** [comment]
The FedRAMP Security Controls Baseline spreadsheet has been updated to remove JAB and PMO references and make all FedRAMP document references uniform and are pending approval.  The following controls were edited:

**Edited Controls:**
- AC-02 (03) H/M
- AC-05 H/M
- AC-08 H/M/L
- AU-02 H/M/L
- AU-06 H/M/L
- AU-06 (06) H
- CA-02 H/M/L
- CA-02 (01) H/M
- CA-02 (03) H/M
- CA-03 H/M/L
- CA-05 H/M/L
- CA-06 H/M/L
- CA-07 H/M/L
- CA-08(2) H/M
- CM-02 H/M/L
- CM-03 H/M
- CP-02 H/M/L
- CP-04 H/M/L
- IR-03 H/M
- MP-05 H/M
- RA-03 H/M/L
- RA-05 H/M/L
- SC-07 H/M/L
- SC-07 (05) M

Waiting on final approval of those changes and timeline estimates before making changes to the FedRAMP OSCAL profiles, and documentation
